components/butterfly/src/rumor/mod.rs

```diff
@@ -51,6 +52,15 @@ use member::Membership;
 pub use protocol::newscast::{Rumor as ProtoRumor, RumorPayload, RumorType};
 use protocol::{FromProto, Message};
 
+lazy_static! {
+    static ref IGNORED_RUMOR_COUNT: CounterVec = register_counter_vec!(
+        "hab_butterfly_ignored_rumor_total",
```

Comment:
You have another gauge “hab_butterfly_peer_health_total” and to avoid confusion, recommend renaming to `*_count` or removing `_total` as this is a counter.

Reply:
I was following https://prometheus.io/docs/practices/naming/ when naming these things.  That page recommends that every metric have a suffix and for counters, the appropriate suffix is `_total`.